packages/web-pkg/src/composables/eventBus/useEventBus.ts

```diff
@@ -0,0 +1,5 @@
+import { eventBus } from '../../services'
+
+export const useEventBus = () => {
```

Comment:
The `index.ts` file lacks an export of this composable (+ imports in the file that uses it can be shortened then)